### ISSUE TITLE
Error in spherical harmonics implemetation: Missing an attribute

### ISSUE THREAD

**Issue by SedighehKarimi** [ITC-Water-Resources/shxarray#1]
**Bug description**

Hello, My prompt is to use the `sh.degvar` function. However, it fails and I encounter an `AttribueError` with the message 'DataArray' has no attribute 't'.

`nmax =300

analysis = tws.sh.analysis(nmax)

sh_coefficients = analysis.compute()

sh_coefficients.sh.degvar()`

The `sh.degvar` function is expected to calculate the variance of spherical harmonic coefficients without errors.

**Comment by strawpants** [comment]
I can reproduce the error with the following standalone code:
 ```
import shxarray
import xarray as xr
nmax=20
nmin=2
daones=xr.DataArray.sh.ones(nmax=nmax,nmin=nmin)
dadv=daones.sh.degvar()
```
This is fixed now in 1864bccea1c85769e18c15ac593af930874f3999

**Comment by SedighehKarimi** [comment]
I have tested the fix, and it is working.

Thanks!
